docs/modules/organization/pages/users/manage-users.adoc

```diff
@@ -92,3 +92,31 @@ When you're finished, select *Save*.
 == View user history
 
 In *User History*, you can view the xref:organization:users/use-the-user-history-report.adoc[user's history report].
+
+== Export all users
+
+You can export user data as a CSV file, making it easy to retrieve a list of all users in your organization for analysis and reporting.
+
+image:export-users-button.png[width=1000,alt="Button to export all users"]
+
+To export a list of all users:
+
+1. Navigate to *Org Management*, then select the *Users* tab.
+2. In the *Users* tab, click on the Export Users button. A CSV file named UserExport.csv will automatically download to your device.
+[NOTE]
+You must be an Org Admin to access the *Users* tab.
```

Comment:
You can't see Org Management in the left-hand nav unless you are an admin. So maybe "You must be an Org Admin to access Org Management."